plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/BaseSqlServerConnectorTest.java

```diff
@@ -213,46 +209,52 @@ public void testColumnComment()
     @Test
     public void testPredicatePushdown()
     {
+        // varchar inequality
+        assertThat(query("SELECT regionkey, nationkey, name FROM nation WHERE name != 'ROMANIA'"))
```

Comment:
Yes it will be also pushdown - it will be turned into discrete set and pushdown allowed.
I will actually modify this test to have NOT IN, to show full functionality.

Reply:
Test modified